### PR TITLE
Fix root CA secret name

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Fix Root CA Secret name

**Related issue(s)**

### CODE REVIEW

File: resources/compass/charts/connector/templates/role-binding.yaml [hunk 1]
```diff
@@ -34,29 +34,29 @@ roleRef:
...
   verbs: ["get"]
 
 ---
 kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.name }}
-  namespace: {{ .Values.global.connector.secrets.rootCA.namespace | default "default" }}
+  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.cacert }}
+  namespace: {{ .Values.global.connector.secrets.rootCA.cacertspace | default "default" }}
```
Comment (franpog859) [inline resources/compass/charts/connector/templates/role-binding.yaml:59]: I think it should still be `namespace`

File: resources/compass/charts/connector/templates/role-binding.yaml [hunk 2]
```diff
@@ -34,29 +34,29 @@ roleRef:
...
-{{ if .Values.global.connector.secrets.rootCA.name }}
+{{ if .Values.global.connector.secrets.rootCA.cacert }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.name }}
-  namespace: {{ .Values.global.connector.secrets.rootCA.namespace | default "default" }}
+  name: {{ template "fullname" . }}-{{ .Values.global.connector.secrets.rootCA.cacert }}
+  namespace: {{ .Values.global.connector.secrets.rootCA.cacertspace | default "default" }}
```
Comment (franpog859) [inline resources/compass/charts/connector/templates/role-binding.yaml:43]: The same thing there

File: resources/compass/charts/connector/templates/role-binding.yaml (hunk 1 above)
Comment (Szymongib) [reply resources/compass/charts/connector/templates/role-binding.yaml]: Yea definitely, got a little to wild with cmd+R xD

**Comment by franpog859** [comment]
/test pre-master-kyma-gke-upgrade
here is the [previous log](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/8007/pre-master-kyma-gke-upgrade/1249953672566149121)

**Pull request merged**